storage/vdi_automaton.ml

```diff
@@ -87,10 +87,10 @@ let ( + ) state operation =
 		| Activated _, Detach     -> error ()
 
 let superstate states =
-	let activated = List.fold_left (fun acc s ->
-		acc || (s = Activated RO) || (s = Activated RW)) false states in
-	let rw = List.fold_left (fun acc s ->
-		acc || (s = Activated RW) || (s = Attached RW)) false states in
+	let activated = List.exists (fun s ->
```

Comment:
Thanks, I updated the pull request, I think this makes the code even more readable.